spec.md

```diff
@@ -371,26 +362,12 @@ authenticated before use.
 ## Policy Validation
 
 When sending to an MX at a domain for which the sender has a valid and
-non-expired SMTP STS policy, a sending MTA honoring SMTP STS SHOULD validate
+non-expired SMTP STS policy, a sending MTA honoring SMTP STS MUST validate
```

Comment:
Do we want to call out the possibility for a sending MTA to be non-STS-aware but to still do offline reporting? Or is that not necessary?